phoenix-core/src/main/java/org/apache/phoenix/compile/PostDDLCompiler.java

```diff
@@ -155,7 +180,14 @@ public boolean hasUDFs() {
                         };
                         PhoenixStatement statement = new PhoenixStatement(connection);
                         StatementContext context = new StatementContext(statement, resolver, scan, new SequenceManager(statement));
-                        ScanUtil.setTimeRange(scan, timestamp);
+                        long ts = timestamp;
```

Comment:
No, also this is the only place where TransactionUtil.convertToNanoseconds(ts) is called.